cli/src/lib/semver.js

```diff
@@ -8,7 +8,7 @@ export type Version = {
   major: number | "x",
```

Comment:
This file as a whole seems to be duplicating some functionality of https://github.com/npm/node-semver which is already a dependency.